src/platform/user/authentication/utilities.js

```diff
@@ -79,12 +72,25 @@ function redirectWithGAClientId(redirectUrl) {
   }
 }
 
+export function standaloneRedirect() {
+  const searchParams = new URLSearchParams(window.location.search);
+  const application = searchParams.get('application');
+  const to = searchParams.get('to');
```

Comment:
I asked this before and it was fine but just confirming that the `to` parameter is validated in some way so that we can't just put anything there - `va.gov?to=malicious-website.com`

Reply:
if no `application` query parameter was provided, the if block on line 81 would fail and null would be returned.

Reply:
Hm what about `a.gov?application=something&to=malicious-website.com`?

Reply:
In that case the url would be `https://ehrm-va-test.patientportal.us.healtheintent.com/malicious-website.com`